src/api/v1/organisations/index.ts

```diff
@@ -17,4 +17,20 @@ export default [
       return '';
     },
   },
+  {
+    method: 'GET',
+    path: '/cls',
```

Comment:
My thinking was that the `api/v1` directory would be organised into directories based on the root level path parameter, so all `/organisations/...` routes would go into and `organisations` directory.

So I think I would either put this in a `cls` directory or think about whether CLS data should be accessible under some (yet to be designed) survey endpoint (such as `/surveys/cls` or `/surveys/answers/cls` or similar).

Reply:
Assuming this is achieved by making each group of endpoints a plugin and using [`options.routes.prefix`](https://hapijs.com/api#server.register())?

Reply:
I was planning on only using the prefix option for the top-level API v1 plugin in order to prefix either `/api/v1` or `/v1` (depending on what URL we host it at) onto all the API routes.

I thought it would be good (for clarity and searchability) to require all route definitions to specify their full path.